website/source/docs/connect/proxies/envoy.md

```diff
@@ -36,7 +36,8 @@ compatible Envoy versions.
 
 | Consul Version | Compatible Envoy Versions |
 |---|---|
-| 1.5.2 and higher | 1.11.1 1.10.0, 1.9.1, 1.8.0† |
+| 1.7.0 and higher | 1.12.2, 1.11.2, 1.10.0, 1.9.1 |
```

Comment:
Just a note that maybe we should clarify in the changelog for 1.7.0 that we now require Envoy > 1.9.1.

Reply:
👍